modules/sdk-coin-sui/src/lib/iface.ts

```diff
@@ -57,11 +137,67 @@ export interface PayTx {
   amounts: number[];
 }
 
-export interface SuiTransaction {
+export interface MoveCallTx {
+  package: SuiObjectRef;
+  module: string;
+  function: string;
+  typeArguments: TypeTag[];
+  arguments: SuiJsonValue[];
+}
+
+export interface RequestAddDelegationTx {
+  coins: SuiObjectRef[];
+  amount: number;
+  validatorAddress: SuiAddress;
+}
+
+export interface SuiTransaction<T = PayTx | MoveCallTx> {
   type: SuiTransactionType;
   sender: string;
-  payTx: PayTx;
+  tx: T;
```

Comment:
Seems to be breaking change?, mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))